pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -159,7 +159,7 @@ type Plugin interface {
 // the timestamp when it's added to the queue or recording per-pod metrics.
 type PodInfo struct {
 	Pod *v1.Pod
-	// The time pod added to the scheduling queue.
+	// The last time when the pod is tried to schedule.
```

Comment:
Should this be added as a separate variable instead of overriding the time when the pod was added to the scheduling queue ? And how would the new logic prioritize between a pod1 that was added to the scheduling queue `n` seconds ago vs a pod2 that was tried scheduling `x` seconds ago, but  was added to the scheduling queue later (`n` > `x`) ?

Reply:
1. This PR does not intend to make the field `Time` better, I just make the comment more clear.
2. In most case, the new prioritize rule does not take the field `Time` into account, because it is the most less significant bits when we comparing, please check:

https://github.com/kubernetes/kubernetes/blob/e64be696d610c0063902c68422556730ed43c785/pkg/scheduler/internal/queue/scheduling_queue.go#L214-L215